src/screens/FeedScreen/Content.js

```diff
@@ -39,9 +39,8 @@ const Content = ({
     onLayoutTopicChip,
     heightTopic,
     amountLineTopic,
-    heightPoll,
-    handleMarginVertical,
-    onPollLayout
+    onPollLayout,
+    heightPoll
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleMarginVertical` function is removed from the destructuring assignment of `useCalculationContent()`. If this function is no longer used in this component, this change is fine. However, if it's still needed somewhere in the component, this could cause a reference error.
```diff
-    heightPoll,
-    handleMarginVertical,
-    onPollLayout
+    onPollLayout,
+    heightPoll
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->